heron/scheduler-core/src/java/com/twitter/heron/scheduler/SubmitterMain.java

```diff
@@ -455,9 +507,20 @@ protected URI uploadPackage(IUploader uploader) throws UploaderException {
   }
 
   protected void callLauncherRunner(Config runtime)
-      throws LauncherException, PackingException {
+      throws LauncherException, PackingException, SubmitDryRunResponse {
     // using launch runner, launch the topology
     LaunchRunner launchRunner = new LaunchRunner(config, runtime);
     launchRunner.call();
   }
+
+  protected String renderDryRunResponse(SubmitDryRunResponse resp) {
+    SubmitDryRunRender render = new SubmitDryRunRender(resp);
+    DryRunFormatType formatType = Context.dryRunFormatType(config);
+    switch (formatType) {
```

Comment:
resolved. see: https://github.com/twitter/heron/pull/1629/files#diff-d46e068b4c18dd1146772b304b784859R456 and https://github.com/twitter/heron/pull/1629/files#diff-1043de1ec8f8e4c0194d4e6035e654cfR519